tests/simulation_tests.ipynb

```diff
@@ -23,11 +23,11 @@
      "name": "stdout",
```

Comment:
The triple quotes will comment out a block of code. I was using this here because I was trying to see if the reason github actions was hanging was because the notebook was running for too long.